optin/puller/workflows/get_account_states.py

```diff
@@ -19,6 +20,40 @@ def get_addresses(filename):
 		return [optin_request.address for optin_request in in_progress_database.requests]
 
 
+async def populate_balances(connection, client, addresses):
+	database = BalancesDatabase(connection)
+	database.create_tables()
+
+	limiter = AsyncLimiter(20, 1.0)
+
+	async def get_state(address):
+		async with limiter:
+			account_balance = await client.historical_balance(address, SNAPSHOT_HEIGHT)
+			database.add_account_balance(*account_balance)
+
+			print('.', end='', flush=True)
+
+	tasks = list(map(get_state, addresses))
+	await asyncio.gather(*tasks)
+
+
+async def populate_multisig(connection, client, addresses):
+	database = MultisigDatabase(connection)
+	database.create_tables()
+
+	limiter = AsyncLimiter(20, 1.0)
+
+	async def get_state(address):
+		async with limiter:
+			account = await client.account(address)
+			database.insert_if_multisig(account)
```

Comment:
hmm, this gets _current_ account state, so is this ok?

Reply:
we do not have historical multisigs, so i guess so